google/resource_compute_subnetwork.go

```diff
@@ -164,8 +184,9 @@ func resourceComputeSubnetworkCreate(d *schema.ResourceData, meta interface{}) e
 	// "When creating a new subnetwork, its name has to be unique in that project for that region, even across networks.
 	// The same name can appear twice in a project, as long as each one is in a different region."
 	// https://cloud.google.com/compute/docs/subnetworks
+	subnetworkV1.Region = region
 	subnetwork.Region = region
-	d.SetId(createSubnetID(subnetwork))
+	d.SetId(createSubnetID(subnetworkV1))
```

Comment:
Should we set the id based on which subnet version (beta or release api) the request was sent to? Or is the id not generated from the api?